package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "radar-sdk-js",
-  "version": "4.1.18",
+  "version": "4.2.0-beta.0",
```

Comment:
Are we shipping these changes as `4.2`?

Reply:
Removing `trackVerifiedToken()` and renaming the error codes are breaking changes, so should prob be `4.2.x` instead of `4.1.x`